ids/node_id.go

```diff
@@ -76,12 +74,6 @@ func ToNodeID(bytes []byte) (NodeID, error) {
 	return NodeID(nodeID), err
 }
 
-func NodeIDFromCert(cert *staking.Certificate) NodeID {
```

Comment:
We have import cycle conflict: `ids -> staking -> secp256k1fx -> ids`
I considered that moving `NodeIDFromCert` out of this package is the least diff and package-structure wise most reasonable change.
Its currently moved to `vms/proposervm` as its the only package that is uses it.